tests/test_resources/test_clusters/test_on_demand_cluster.py

```diff
@@ -183,11 +183,6 @@ def test_fn_to_docker_container(self, ondemand_aws_cluster):
 
     @pytest.mark.level("minimal")
     def test_status_scheduler_basic_flow(self, cluster):
```

Comment:
I think we can delete this test because there is a similar test in test_cluster.py, and the on-demand cluster test suite inherits from the cluster test suite. In the cluster test suite, test_status_scheduler_basic_flow is marked as local, but I think that minimal tests are running local tests as well.

@jlewitt1 wdyt?